functional-tests/test-bed/src/test/java/org/arquillian/smart/testing/ftest/customAssertions/BuildProjectAssert.java

```diff
@@ -0,0 +1,59 @@
+package org.arquillian.smart.testing.ftest.customAssertions;
+
+import java.io.File;
+import org.assertj.core.api.AbstractAssert;
+import org.assertj.core.api.FileAssert;
+import org.jboss.shrinkwrap.resolver.api.maven.embedded.BuiltProject;
+
+public class BuildProjectAssert extends AbstractAssert<BuildProjectAssert, BuiltProject> {
+
+    BuildProjectAssert(BuiltProject actual) {
+        super(actual, BuildProjectAssert.class);
+    }
+
+    public static BuildProjectAssert assertThat(BuiltProject module) {
+        return new BuildProjectAssert(module);
+    }
+
+    public BuildProjectAssert hasAllBuiltSubModulesContainEffectivePom(String effectivePom) {
+        actual.getModules().forEach(subModule -> assertThat(subModule).containsEffectivePom(effectivePom));
+        return this;
+    }
+
+    public BuildProjectAssert hasAllBuiltSubModulesContainReport(String report) {
```

Comment:
it is not that all submodules - just the relevant ones right? It would be good to either change the method name or explain it in javadoc... or both